examples/myproject/nhost/nhost.toml

```diff
@@ -26,7 +26,7 @@ liveQueriesMultiplexedRefetchInterval = 3000
 version = 18
 
 [auth]
-version = '0.21.2'
+version = '0.29.6'
```

Comment:
shouldn't this be '0.30.0' ?

Reply:
doesn't really matter but changing nonetheless